.editorconfig

```diff
@@ -20,4 +20,5 @@ ktlint_standard_max-line-length = 160
 ktlint_standard = enabled
 
 [{*.py, *.pyi}]
-indent_size = 4
\ No newline at end of file
+indent_size = 4
```

Comment:
kinda messy getting both edits in this one PR 😢

Reply:
Yes, this is the problem with how GitHub handles (or pretty much doesn't handle) stacked commits. :(